scss/exercises.module.scss

```diff
@@ -0,0 +1,11 @@
+.exerciseList__container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(320px, 1fr));
+  gap: 2em;
+}
+
+.exerciseList__solveExercisesButtonContainer {
+  @media (max-width: 767px) {
+    width: 100%;
+  }
+}
```

Comment:
Unfortunately there's no way to use CSS grid with Bootstrap CSS classes and there's no Bootstrap class like `w-*-100`  (where `*` is xs, sm, md, lg, xl), so we have to do this in SCSS.

Reply:
Looks good! Can we keep the font size/height of the button the same across all breakpoints? It looks really small and it can be hard to notice if someone is not familiar with it.

Reply:
Yeah, I can make it so it's always the same size across all breakpoints.

Reply:
It looks like this change might take a bit more work, so I'll make the button the same size across breakpoints in a separate pull request.

Reply:
I made an issue for this https://github.com/garageScript/c0d3-app/issues/2343